config/routes.rb

```diff
@@ -9,14 +9,15 @@
   post '/login', to: "sessions#create"
   get '/logout', to: "sessions#destroy"
 
-  get '/profile', to: "users#show"
-  patch '/profile', to: "users#update"
-  get '/profile/edit', to: "users#edit"
+  get '/profile', to: "users#show" # to-do: move to "user/users#show" using the profile scope / user module
```

Comment:
Great notes, thank you!

Reply:
Yeah, I thought about changing it here, but that'd involve moving around 6+ files, so maybe we can knock those out in a separate branch.